homeassistant/components/sanix/sensor.py

```diff
@@ -0,0 +1,148 @@
+"""Platform for Sensor integration."""
```

Comment:
Please only have 1 platform in the initial PR

Reply:
I left the Sensor platform as the only one for the initial PR.